src/components/PostInfo/PostInfo.jsx

```diff
@@ -1,3 +1,29 @@
-export const PostInfo = () => (
-  <>Put the post here</>
+import './PostInfo.scss';
+import { UserInfo } from '../UserInfo/UserInfo';
+import { CommentList } from '../CommentList';
+
+export const PostInfo = ({ post }) => (
+  <div className="PostInfo">
+    <div className="PostInfo__header">
+      <h3 className="PostInfo__title">{post.title}</h3>
+
+      <p>
+        { 'Posted by '}
+        <UserInfo user={post.user} />
+      </p>
+    </div>
+
+    <p className="PostInfo__body">
+      {post.body}
+    </p>
+
+    <hr />
+
+    <CommentList comments={post.comments} />
```

Comment:
you invoking a <CommentList /> component BEFORE checking, is post.comments is empty 

better practice: 

```
     {isNotEmptyArray ? (
        <CommentList comments={commentsList} />
      ) : (

     <hr />
        <b data-cy="NoCommentsMessage">No comments yet</b>

      )}
```